.travis.yml

```diff
@@ -1,11 +1,22 @@
 language: java
 
-# only java 8 supported for now
-jdk: oraclejdk8
+# test with java 9 support
+jdk: 
+- openjdk11
```

Comment:
JDK8 should still be tested, many still use it AFAIK. So I think it should be part of the list